### PR TITLE
Additional information on PinkNoise as gathered in #2678.

### PR DESCRIPTION
**Pull request by LFSaw**
#2678

### CODE REVIEW

**Comment by brianlheim** [comment]
Thanks for this @LFSaw. Will review tonight.

File: HelpSource/Classes/PinkNoise.schelp [hunk 1]
```diff
@@ -6,9 +6,15 @@ categories::  UGens>Generators>Stochastic
...
-This gives equal power over the span of each octave.
-This version gives 8 octaves of pink noise.
+Generates noise whose spectrum falls off in power by 3 dB per octave, which gives equal power over the span of each octave.
+This version is band-limited to 8 octaves.
+
+Internally, this UGen calculates its output by  means of the Voss-McCartney algorithm.
+link::http://www.firstpr.com.au/dsp/pink-noise/allan-2/spectrum2.html::
+
+note::
+The values produced by this UGen were observed to lie with very high probability between approx. -0.65 and +0.81 (before being multiplied by mul). The signal's RMS is approximatly -16 dB.
```
Comment (brianlheim) [inline HelpSource/Classes/PinkNoise.schelp]: Could we change both "approx." and "approximatly" to "approximately"?

**Comment by brianlheim** [comment]
@LFSaw this looks good to me, but @telephon were you also planning on doing something with the examples like you mentioned in #2678 ?

**Comment by brianlheim** [comment]
@snappizz We were still waiting on changes here...

**Comment by LFSaw** [comment]
but merged it is now?

**Comment by snappizz** [comment]
It has been reverted. While I concede that I merged prematurely, I don't think we should be perfectionistic concerning help PRs, esp. when it comes to typos.

**Comment by brianlheim** [comment]
> I don't think we should be perfectionistic concerning help PRs, esp. when it comes to typos.

I respectfully disagree; we should get it right the first time. It's going to need to be corrected at some point, and the best place to correct it is in the place where the typo was made in the first place.

**Comment by snappizz** [comment]
> It's going to need to be corrected at some point

No, I don't think typos *need* to be corrected unless they inhibit clarity or are otherwise exceptionally bad.

Slicing this as a cost-benefit analysis: there is a cost to requesting a change in a PR, and that is the effort required to communicate and fix the issue and the time spent waiting for that process to unfold. In the case of typos in help files, the utility gained from having a typo fixed is very small, and I argue not enough to justify that cost.

Fixing typos is a suboptimal use of developer time. If you want to spend your time that way, then that's your prerogative, but I'd prefer if we don't ask others to do it. If the user has allowed contributors with write access to push to the PR branch, it's best to just fix the typo yourself.

**Comment by brianlheim** [comment]
This is getting unnecessarily hostile. I do in fact care a lot about the quality of public-facing documentation, and I really don't appreciate the implication that I'm wasting my time or others' by doing so. What's most efficient in terms of workflow is definitely worth discussing.

**Comment by snappizz** [comment]
Sorry if the last message was too strongly worded! I just think it's quicker to either fix it yourself or fix it after merge than to ask others to fix it...

**Comment by LFSaw** [comment]
I'm soory but I don;t understand the situation anymore here. 
I did a PR and waited for comments/merges.
I did overlook the approx./approximately suggestion and therefore did not "fix" it.
Things got merged, I asked if they are merged, I removed my branch in my repo.

What is the status here now?

**Comment by brianlheim** [comment]
@LFSaw You're fine, don't worry about it :)

**Comment by brianlheim** [comment]
It's merged; the typo is still there, in master now, but it's not a big deal.

**Comment by vivid-synth** [comment]
@brianlheim if you want to do another PR fixing the typos, I'm also in favor of clean docs

**Pull request merged**